src/runScript.js

```diff
@@ -16,13 +16,13 @@ module.exports = function runScript(commands, pathsToLint, packageJson, config)
 
   const filePathChunks = chunk(pathsToLint, calcChunkSize(pathsToLint, chunkSize))
 
-  const lintersArray = Array.isArray(commands) ? commands : [commands]
+  const commandsArray = Array.isArray(commands) ? commands : [commands]
```

Comment:
Should we enforce to always use Array in the advanced config and do the transform in ` getConfig`?